macros/cross-adapter-modeling/sessionization/segment_web_user_stitching.sql

```diff
@@ -0,0 +1,36 @@
+{% macro segment_web_user_stitching() %}
+
+    {{ adapter_macro('segment.segment_web_user_stitching') }}
+
+{% endmacro %}
+
+
+{% macro default__segment_web_user_stitching() %}
+
+{{config(materialized = 'table')}}
+
+with events as (
+
+    select * from {{ref('segment_web_page_views')}}
+
+),
+
+mapping as (
+
+    select distinct
+    
+        anonymous_id, 
+        
+        last_value(user_id) over (
```

Comment:
Should this be `last_value(user_id ignore nulls)`?

Reply:
this is a great catch. I actually found this bug in another client just this morning because it was causing massive problems. impressive that you found this in a code review!